Source/inv.cpp

```diff
@@ -897,9 +897,13 @@ void CheckInvCut(int pnum, Point cursorPosition, bool automaticMove, bool dropIt
 	}
 
 	if (dropItem && !holdItem.isEmpty()) {
-		if (IsStashOpen && AutoPlaceItemInStash(player, holdItem, true)) {
-			holdItem._itype = ItemType::None;
-			NewCursor(CURSOR_HAND);
+		if (IsStashOpen) {
+			if (AutoPlaceItemInStash(player, holdItem, true)) {
+				holdItem._itype = ItemType::None;
+				NewCursor(CURSOR_HAND);
+			} else {
+				player.SaySpecific(HeroSpeech::IHaveNoRoom);

```

Comment:
For variation, also "I" can be taken as referring to the hero, and not the stash which is Gillians :)
```suggestion
				player.SaySpecific(HeroSpeech::WhereWouldIPutThis);
```